packages/test/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@ceramicnetwork/test",
+  "version": "0.1.0",
+  "description": "",
```

Comment:
Why is this a separate package? Couldn't it just be tests in the account package?

Reply:
This is a dummy package that won't end up in the final PR, but I'm using it to act as a reference implementation of a package that would use ceramic-account.